examples/ai-agent/package.json

```diff
@@ -25,8 +25,11 @@
   },
   "dependencies": {
     "@ai-sdk/openai": "^0.0.66",
+    "@hono/node-server": "^1.13.7",
+    "@hono/node-ws": "^1.0.4",
     "@rivetkit/react": "workspace:*",
     "ai": "^4.0.38",
```

Comment:
upgrade to v5. i think it changes how chat persistence works too. make sure to check their changelog.